hdfs/client.py

```diff
@@ -661,7 +661,6 @@ def _download(_path_tuple):
             _writer.write(chunk)
 
     # First, we figure out where we will download the files to.
-    hdfs_path = self.resolve(hdfs_path)
```

Comment:
@pprett - when do you think you will have some time to edit this PR? (If it's OK with you, I could also do the changes.)